leaflet/leaflet-tests.ts

```diff
@@ -427,6 +427,23 @@ L.Control.include({});
 L.Control.mergeOptions({});
 L.Control.addInitHook(() => {});
 
+let icon = L.divIcon({
+    className: 'css-icon',
+    html: '<i id="marker_icon" class="marker"></i>',
+    iconSize: [32, 32],
+    iconAnchor: [0, 24],
+    popupAnchor: [0, -36]
+}); 
+
+// Icon > DefaultIcon > DivDicon
+// Option object should be nullable
+L.marker([2,2], { icon: icon })
+	.bindPopup("Hello Leaflet"); //string should be possible 
+
+// it should be possible to define the default path without casting
+// (<any> L.Icon.Default)
+L.Icon.Default.imagePath = "/any/valid/uri/or/path"; 
```

Comment:
From the documentation:

> In order to customize the default icon, just change the properties of L.Icon.Default.prototype.options (which is a set of Icon options).

As I said, `Default` is another class, not an instance.